server/backend/sync/memory/pubsub.go

```
@@ -113,34 +113,33 @@
 		)
 	}
 
 	sub.Close()
 
-	if subs, ok := m.subscriptionsMapByDocRefKey[documentRefKey]; ok {
+	if subs, ok := m.subscriptionsMap.Get(docKey); ok {
 		subs.Delete(sub.ID())
 
 		if subs.Len() == 0 {
-			delete(m.subscriptionsMapByDocRefKey, documentRefKey)
+			m.subscriptionsMap.Delete(docKey, func(subs *Subscriptions, exists bool) bool {
```

golangci-lint

[warning] unused-parameter: parameter 'subs' seems to be unused, consider removing or renaming it as _

(revive)